sycl/include/CL/sycl/detail/scheduler/commands.h

```diff
@@ -408,11 +408,11 @@ template <int DimSrc, int DimDest> class CopyCommand : public Command {
 // accessor in lambda function. 'ClKernel' is the kernel. 'HostKernel'
 // is the pointer to the lambda function.
 template <int AccessDimensions, typename KernelType>
-uint passGlobalAccessorAsArg(uint I, int LambdaOffset, cl_kernel ClKernel,
-                             const KernelType &HostKernel);
+uint32_t passGlobalAccessorAsArg(uint I, int LambdaOffset, cl_kernel ClKernel,
```

Comment:
Are you going to replace `uint I`?

Reply:
Yes, thank you. Sorry, I did 'git commit' without doing 'git add', that is why uint -> uint32_t for this parameter 'I' was not updated here 20 minutes ago. It is fixed now.